crates/eip5792/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "alloy-eip5792"
+description = "Types for the 'wallet' Ethereum JSON-RPC namespace"
+
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = [
+    "-Zunstable-options",
+    "--generate-link-to-definition",
+    "--show-type-layout",
+]
+
+[dependencies]
+alloy-primitives = { workspace = true, features = ["serde", "map"] }
+alloy-serde.workspace = true
```

Comment:
this we can't do, 
or at leat we need extract the serde crate first:

https://github.com/alloy-rs/serde